src/linear_algebra/basic.lean

```diff
@@ -224,6 +224,18 @@ add_monoid_hom.map_sum ⟨@to_fun R M M₂ _ _ _ _ _, rfl, λ x y, rfl⟩ _ _
 instance : monoid (M →ₗ[R] M) :=
 by refine {mul := (*), one := 1, ..}; { intros, apply linear_map.ext, simp {proj := ff} }
 
+@[simp] lemma mul_apply (f g : M →ₗ[R] M) (m : M) :
```

Comment:
Would this be better as `coe_mul` without `m`?

Reply:
My instinct is to be conservative about non-fully-applied simp lemmas, until there's a need for them. (I like my easy proofs to look like `ext, simp`...) But I can't articulate any particular objection here, beyond that it's more awkward to state `coe_mul` than it is to state `mul_apply`!

Reply:
I think it would be to good to have the lemma, even if you'd prefer not to mark it simp

Reply:
And we already have `ring_hom.coe_pow` in this style

Reply:
Done.